include/dlaf/interface/cholesky.h

```diff
@@ -0,0 +1,105 @@
+//
```

Comment:
I would definitely separate out the actual C interface functions into a separate header so that they can actually be included in C files without dragging in all of dlaf and pika (which won't work anyway).